charts/internal/machineclass/values.yaml

```diff
@@ -20,3 +20,7 @@ machineClasses:
   credentialsSecretRef:
     name: cloudprovider
     namespace: shoot-namespace
+# reservationIDs:
+# - res1
+# - res2
+# onlyReserved: false
```

Comment:
I think `reservedOnly` or maybe even `reservedDevicesOnly` would be better.

Reply:
That means changing all of the areas it is produced or consumed, sorry.

Reply:
No issues, I like it, definitely better :)
Done in 58d2278007312002fb12242173bdcfee68305460.